src/main/java/frc/robot/subsystems/scoring/ShooterIOSim.java

```diff
@@ -41,21 +54,36 @@ public void setKickerVolts(double volts) {
 
     @Override
     public void updateInputs(ShooterIOInputs inputs) {
-        shooterSim.update(Constants.loopTime);
+        shooterLeftSim.update(Constants.loopTime);
+        shooterRightSim.update(Constants.loopTime);
+
+        shooterLeftAppliedVolts =
+                shooterFeedforward.calculate(shooterLeftSim.getAngularVelocityRadPerSec())
+                        + shooterLeftController.calculate(
+                                shooterLeftSim.getAngularVelocityRadPerSec(),
+                                shooterLeftGoalVelRPM * 2.0 * Math.PI / 60.0);
```

Comment:
Could we add a constant for converting RPM to rad/s (similar to `kRadiansPerSecondtoRPM`)?

(would also apply below)